deploy/deploy-centralvenv.sh

```diff
@@ -0,0 +1,309 @@
+#!/bin/bash
+
+### Usage: Usage: deploy-centralvenv.sh -t <wmcore_tag> -c <central_services_url> [-v wmcore_path] [-p  <patches>] [-s  <service_names>] [-l  <component_list>]
+### Usage:
+### Usage:   -t  <wmcore_tag>       WMCore tag to be used for this deployment [Default: 2.0.0]
+### Usage:   -c  <central_services> Url to central services (e.g. tivanov-unit01.cern.ch)
```

Comment:
Not sure about that  one. I was keeping it as a mandatory flag as a precaution from somebody not hitting an already deployed service by overlooking on what and where he is currently deploying.

Reply:
Coming back to this comment here. 
I am not sure how to proceed actually. Because, in reality, this variable is not used anywhere yet in the current script. Back then, when we were using the `cmsweb` scripts for deploying in a virtual environment, we were having the `BASE_URL` provided from the setup environment : [[1]](https://github.com/dmwm/deployment/blob/541068546b79b2646d021a3a7f81ddfe60f7e772/reqmgr2/config.py#L13), while we now have the link hard coded in the service configs depending on the configuration branch to be used [[2]](https://gitlab.cern.ch/cmsweb-k8s/services_config/-/blob/preprod/reqmgr2/config.py#L14). At the same time we would need to have the ability to setup outside those K8 clusters - for development and debugging or any other needs, so the `BASE_URL` will need to be configurable at deployment time (regardless of the config branch to be used). One possible approach would be a `sed` command over all the config files, but this I'd consider as really far from elegant solution. @vkuznet  @amaltaro any ideas  are very welcome. Thanks!
 
[1]
https://github.com/dmwm/deployment/blob/541068546b79b2646d021a3a7f81ddfe60f7e772/reqmgr2/config.py#L13 

[2]
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/blob/preprod/reqmgr2/config.py#L14

Reply:
I think what you need to do is the following. Instead of central FE URL, you should allow user to specify configs to use. This approach is more flexible since I can have my local configs pointing to my local services, or I can have a config with mix URLs (some local and some central). So far you **force** users to use central service and I'm advising you to stop doing that since as developer I don't need central service and can run all things at place I want to (I can choose to use central FE or I can run local services).

Reply:
Well I have already answered this few comments bellow, but just to mention: the parameter is not mandatory any more, and the capability of providing user's configs is there.